build.gradle

```diff
@@ -12,31 +12,40 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
-}
+    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'org.asynchttpclient:async-http-client:2.8.1'
+
+    compile "org.slf4j:slf4j-api:1.6.1"
```

Comment:
Do we need these libraries? I'm not sure. Try removing these and see if everything still works!

Reply:
No we don't, Ill remove them